Description: forbid installing npm
 It does not break anything per se, but /usr/local/bin/npm then
 takes precedence, so global config path is /usr/etc/npmrc (found
 by npm default config, relative to /usr/bin/node).
 Upstream does not want to patch this.
Forwarded: not-needed
Bug-Upstream: https://github.com/isaacs/npm/issues/issue/533
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2011-04-30
--- a/lib/install.js
+++ b/lib/install.js
@@ -330,7 +330,14 @@
   function cb (er) {
     return cb_(er, [[targetFolder, target._id]])
   }
-
+  
+  if (target.name == "npm" && npm.config.get("global")) {
+    var er = new Error("'install npm' breaks debian npm package, use aptitude")
+    er.errno = npm.ELIFECYCLE
+    er.pkgid = target._id
+    return cb(er)
+  }
+  
   chain
     ( [checkEngine, target]
     , [checkCycle, target, previously]
